pkg/networkutils/networkutils_test.go

```diff
@@ -63,6 +63,35 @@ func TestIsIPInUseFail(t *testing.T) {
 	g.Expect(res).To(gomega.BeTrue())
 }
 
+func TestIsPortInUsePass(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	g := gomega.NewWithT(t)
+
+	client := mocks.NewMockNetClient(ctrl)
+	client.EXPECT().DialTimeout(gomock.Any(), gomock.Any(), gomock.Any()).
```

Comment:
We know what the expected values are supposed to be, so let's use them instead of `gomock.Any()`

Here, it should expect `"tcp", address, 500*time.Millisecond`